### ISSUE TITLE
skipif_ocp_version breaking test colllection with internal error

### ISSUE THREAD

**Issue by petr-balogh** [red-hat-storage/ocs-ci#3115]
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/13314/console

```INTERNALERROR>     if skipif_ocp_version(skip_condition[0]):
INTERNALERROR>   File "/home/jenkins/workspace/qe-deploy-ocs-cluster/ocs-ci/ocs_ci/utility/utils.py", line 2079, in skipif_ocp_version
INTERNALERROR>     skip_this = skip_this and eval(comparision_str)
INTERNALERROR>   File "<string>", line 1
INTERNALERROR>     <4.6
INTERNALERROR>     ^
INTERNALERROR> SyntaxError: invalid syntax```

@Avilir  can you please take a look?

**Comment by petr-balogh** [comment]
@Avilir https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/13354/ it affect also upgrade here so I suggest to revert the change if you don't have the fix already in place which really works.

**Comment by Avilir** [comment]
This basic issue fixed by #3117 but still need to be better implemented.

**Comment by Avilir** [comment]
Current get_ocp_version is taking information from the configuration file and not form the actual cluster which can lead to problem on existing cluster
